### PR TITLE
Add GPS DOP debug

### PR DESCRIPTION
**Pull request by haslinghuis**
Adds GPD_DOP debug

https://github.com/betaflight/betaflight/pull/11912

### CODE REVIEW

File: js/flightlog_fields_presenter.js
Comment (KarateBrot) [inline js/flightlog_fields_presenter.js:708]:
```suggestion
                    'debug[all]': 'GPS Dilution of Precision',
                    'debug[0]': 'Number of Satellites',
                    'debug[1]': 'pDOP (positional - 3D)',
                    'debug[2]': 'hDOP (horizontal - 2D)',
                    'debug[3]': 'vDOP (vertical - 1D)',
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]
Thanks for your work! Can we scale down the displayed DOP values by 100? (1 decimal place)
(They get multiplied by 100 in BF because they are originally float values from ).

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [reply js/flightlog_fields_presenter.js]: Thanks. Fixed.

File: js/flightlog_fields_presenter.js
Comment (KarateBrot) [inline js/flightlog_fields_presenter.js:707]:
```suggestion
                    'debug[2]': 'hDOP (horizontal - 2D)',
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [reply js/flightlog_fields_presenter.js]: Fixed

**Review by KarateBrot — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
